csi/main.py

```diff
@@ -41,6 +43,21 @@ def mount_storage():
         logging.info(logf("Volume is mounted successfully", hvol=hvol))
     return
 
+def reconfigure_mounts():
```

Comment:
Single handler expects a func should accept two args, can you pls change the signature to `reconfigure_mounts(*args)`?

Reply:
Ack